dev/containers/userChat.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import io from 'socket.io-client';
+
+const socket = io('http://localhost:3000');
+
+// chat containers will be connected to redux store when data is seeded
+// import { connect } from 'react-redux';
+
+// for testing purposes since were going to leave email the same as barber chat
+// for production we want to extrapolate barber's email an onclick event
+// and pass that email in as data so we know what room to join
+class UserChat extends Component {
```

Comment:
PascalCase not camelCase for component names and file names please :) I thought we were going with calling them Client instead of User? 

Maybe not right this moment, but consider refactoring UserChat and BarberChat in the future to be a single component created twice with maybe a different state to denote whether the attached user is a Client or a Barber.